doc/classes/VisualServer.xml

```diff
@@ -961,7 +961,19 @@
 			<return type="void" />
 			<argument index="0" name="rid" type="RID" />
 			<description>
-				Tries to free an object in the VisualServer.
+				Destroys an object created by the VisualServer. If the [RID] passed is not one created by the VisualServer, an error will be sent to the console.
+				Note: After freeing the object, the RID now has a reference to invalid memory. It is not safe to free an invalid RID. Make sure to reset it before using it again by assigning it to `RID()`.
```

Comment:
```suggestion
				[b]Note:[/b] After freeing the object, the RID now has a reference to invalid memory. It is not safe to free an invalid RID. Make sure to reset it before using it again by assigning it a new RID.
```

Reply:
I specifically meant `assigning it to RID()` as shown in the example shown in the xml. 
eg. 
```
VisualServer.free(myrid)
myrid = RID()
```

Reply:
Yeah but that's not portable syntax (won't work in C#), so I suggested the rephrasing to work it around. Isn't "a new RID" explicit enough? Also, wouldn't it work to assign a new _valid_ RID from another object, if that's what the user wants?

Reply:
`Assigning it to a new RID` to me suggests the meaning, "assigning it to another valid RID you have from another object, or creating a new RID (as part of a new object)". What we specifically want is to re-initialize the variable to a null or zero state. 

The example shown is in GDScript. Aren't all document examples in gdscript? We could put in a C#  version, but this would be the first time I've seen C# going into the manual. I guess the tutorials have both.

I've done the [b] formatting. I'll leave the `RID()` until there is a better alternative.

Reply:
I'm changing it to:
```
[b]Note:[/b] After freeing the object, the RID now has a reference to invalid memory. It is not safe to use or free an invalid RID. Before using the RID again, make sure to assign it to `RID()` or any other valid RID.
[codeblock]
...
```